doc/classes/Object.xml

```diff
@@ -59,6 +59,21 @@
 				Sets a property. Returns [code]true[/code] if the [code]property[/code] exists.
 			</description>
 		</method>
+		<method name="_to_string">
+			<return type="String">
+			</return>
+			<description>
+				Returns a [String] representing the object.
+			</description>
+		</method>
+		<method name="to_string">
+			<return type="String">
+			</return>
+			<description>
+				Returns a [String] representing the object. Default is [code]"[ClassName:RID]"[/code].
+				Override this method with [code]_to_string[/code] to customize the [String] representation. 
```

Comment:
Use `[method _to_string]` to refer to a method with an hyperlink. Also there's a trailing space on this sentence.